bftengine/src/bftengine/ClientReplyMsg.cpp

```diff
@@ -9,14 +9,15 @@
 #include <string.h>
 #include "ClientReplyMsg.hpp"
 #include "assertUtils.hpp"
+#include "ReplicaConfigSingleton.hpp"
 
 namespace bftEngine
 {
 	namespace impl
 	{
 
 		ClientReplyMsg::ClientReplyMsg(ReplicaId primaryId, ReqId reqSeqNum, ReplicaId replicaId)
-			: MessageBase(replicaId, MsgCode::Reply, maxExternalMessageSize) 
+			: MessageBase(replicaId, MsgCode::Reply, bftEngine::ReplicaConfigSingleton::GetInstance().GetMaxExternalMessageSize())
```

Comment:
I don't believe you need the `bftEngine` namespace qualifier, since it's an enclosing namespace. I'm fine leaving it, but removing it would make these lines shorter. Singleton indicates that it's a global type anyway.

Reply:
Thanks for the review!